test/unit/modules/clipboard.js

```diff
@@ -31,6 +31,26 @@ describe('Clipboard', function () {
         }, 2);
       });
 
+      it('paste multiline text', function (done) {
+        this.quill.setSelection(0, 8);
+
+        const captureData = {
+          ...this.clipboardEvent,
+          clipboardData: {
+            ...this.clipboardData,
+            getData: (type) => (type === 'text/html' ? '1<br>' : '1\n'),
+          },
+        };
+
+        this.quill.clipboard.onCapturePaste(captureData);
+        setTimeout(() => {
+          expect(this.quill.root).toEqualHTML(
+            '<p>1</p><p>8</p>',
```

Comment:
If line break is not pasted, a result will be like `<p>18</p>`?

Reply:
yes